lib/kitchen/verifier/shell.rb

```diff
@@ -36,16 +36,23 @@ class Shell < Kitchen::Verifier::Base
       default_config :command, "true"
       default_config :shellout_opts, {}
       default_config :live_stream, $stdout
+      default_config :remote, false
 
       # (see Base#call)
       def call(state)
         info("[#{name}] Verify on instance=#{instance} with state=#{state}")
         sleep_if_set
         merge_state_to_env(state)
-        shellout
+        instance.transport.connection(state) do |conn|
+          conn.execute(run_command)
```

Comment:
run_command looks like return String or shellout(object).  run_command should be returns String I think.

And, conn.execute will always run. This will cause unexpected behavior.